_docs/geyser/faq.md

```diff
@@ -99,6 +99,14 @@ Yes, you can use Geyser as a plugin or a mod for your Minecraft server software
 ## Can I use Geyser with Ngrok?
 Unfortunately, Ngrok is TCP-only, so you will not be able to use Geyser with Ngrok. As an alternative, you can use [playit.gg](https://playit.gg), which supports both TCP and UDP.
 
+## How do I setup [Cosmic Guard](https://cosmicguard.com/)?
+1) You will need access to the host systems terminal. Note: This guide is for Linux systems only!
+2) Setup a "Guard" select gaming then Minecraft: Bedrock edition.
+3) Click setup for linux and run the guardian installer and start the service.
+4) Run "guardian status" and take note of the "Local IP" it starts with "10.31.x.x".
+5) Go to the Geyser spigot config.yml and update the address to the Local IP which you found above.
```

Comment:
Maybe general Geyser instead of specifically the spigot variant? Aka "Geyser config.yml"